### PR TITLE
Add image resource

### LINKED ISSUE #1497: [Add] Image Resource
### Site URL

https://stock.adobe.com/in/

### Description of the above provided site

This site includes a collection of stock photos, stock photography & royalty-free images.

### Category

Frontend

### Sub Category

images

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [X] I am a GSSoC'23 contributor

### PR DESCRIPTION
**Pull request by Sriparno08**
## Fixes Issue

Closes #1497 

## Changes proposed

I have added a new image resource to the website.

## Screenshots

image

## Note to reviewers

This PR is part of GSSOC'23.

### CODE REVIEW

File: database/frontend/images.json [hunk 1]
```diff
@@ -145,5 +145,12 @@
     "url": "https://www.dreamstime.com/",
     "category": "frontend",
     "subcategory": "images"
+  },
+  {
+    "name": "Adobe Stock",
+    "description": "Adobe Stock includes a wide range of stock photos, stock photography & royalty-free images.",
```
Comment (CBID2) [inline database/frontend/images.json:151]:
```suggestion
    "description": "Adobe Stock includes a wide range of stock photos, stock photography,& royalty-free images.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Just found a typo that needs fixing @Sriparno08

**Comment by Sriparno08** [comment]
Fixed it, @CBID2!

**Review by CBID2 — APPROVED** [review]
Well done

**Review by k-deepak04 — APPROVED** [review]

**Comment by Sriparno08** [comment]
Hey @CBID2 @k-deepak04, can you please assign the labels to this PR for GSSoC'23?

**Pull request merged**